tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/CheckAllTool/PackageCheckTool.cs

```diff
@@ -37,9 +37,9 @@ public override Command GetCommand()
             Command command = new("run-checks", "Run validation checks for SDK packages");
             command.AddOption(SharedOptions.PackagePath);
             
-            var checkTypeOption = new Option<string>(
+            var checkTypeOption = new Option<PackageCheckType>(
```

Comment:
re- Sameeksha's comment, do we want a default value here?

Reply:
I guess if it defaults to all that could make sense. Can't think of any other sane default value, but also running all checks by default doesn't feel great since that could take a long time (so perhaps should be opt-in).

Generally I think of flags starting with `--` as optional; having one as required feels weird. If we require people specify the check I would rather see it as a subcommand e.g. `azsdk check changelog`